.github/workflows/release-docker-image.yml

```diff
@@ -88,13 +105,39 @@ jobs:
         with:
           ref: ${{ github.sha }}
 
-      - name: Get semver version from pom and store major and minor version as deptrack version
-        run: echo "DEPTRACK_PROJECT_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -q -DforceStdout | cut -d '-' -f 1 | cut -d '.' -f 1-2).x" >> $GITHUB_ENV
-
       - name: Generate SBOM for maven dependencies
         run: mvn org.cyclonedx:cyclonedx-maven-plugin:makeAggregateBom
 
-      - name: Push SBOM to dependency track
+      - name: Generate SBOM for application Docker image
+        uses: anchore/sbom-action@v0
+        with:
+          image: ghcr.io/puzzle/unilu-pruefungsabfrage:${{ needs.create-release.outputs.semanticVersion }}.${{ needs.create-release.outputs.shortCommitHash }}
+          output-file: application-sbom.xml
+          format: cyclonedx
+          upload-artifact: false
+
+      - name: Generate SBOM for Shibboleth SP Docker image
+        uses: anchore/sbom-action@v0
+        with:
+          image: ghcr.io/puzzle/unilu-pruefungsabfrage-shibboleth-sp:${{ needs.create-release.outputs.semanticVersion }}.${{ needs.create-release.outputs.shortCommitHash }}
+          output-file: shibboleth-sp-sbom.xml
+          format: cyclonedx
+          upload-artifact: false
+
+      - name: Install cyclonedx-cli
+        uses: sigoden/install-binary@v1
```

Comment:
Do we wanna use this action?
I dont know if it will be maintained in the future since it only has 12 commits:
https://github.com/sigoden/install-binary

Reply:
That's a good point. I've switched to https://github.com/supplypike/setup-bin, which seems to have been maintained for at least the last 4 years by multiple people.